includes/Admin/Admin_Page.php

```diff
@@ -300,4 +302,28 @@ public function admin_footer() {
 	public function get_hook_suffix() {
 		return $this->hook_suffix;
 	}
+
+	/**
+	 * Gets default check categories to be selected.
+	 *
+	 * @since 1.0.2
+	 *
+	 * @return string Categories separated by double underscores.
+	 */
+	public function get_default_check_categories_to_be_selected() {
+		$default_check_categories = array(
+			'plugin_repo',
+		);
+
+		/**
+		 * Filters the default check categories to be selected.
+		 *
+		 * @since 1.0.2
+		 *
+		 * @param array $default_check_categories An array of categories.
+		 */
+		$default_categories = (array) apply_filters( 'wp_plugin_check_default_categories', $default_check_categories );
+
+		return ! empty( $default_categories ) ? implode( '__', $default_categories ) : '';
```

Comment:
No need for the `empty` check. `implode` already returns an empty string when the array is empty.